article/app/views/fragments/photoEssayBody.scala.html

```diff
@@ -36,7 +36,7 @@
                         <div class="content__article-body from-content-api js-article__body"
                              itemprop="@if(article.tags.isReview){reviewBody} else {articleBody}"
```

Comment:
Can photo essays be reviews @blongden73 ?

Reply:
nope. :-)